basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,182 @@
 const helpers = require('./project-1');
 
 // start testing!
+// are equal
+describe( 'areEqual(x, y)', function ()
+{
+    it( 'should return true if the arguments are equal', function ()
+    {
+        expect( helpers.areEqual( 15, 15 ) ).toBe( true );
+        expect( helpers.areEqual( 90, 50 ) ).toBe( false );
+        expect( helpers.areEqual( 'test', 'test' ) ).toBe( true );
+    } );
+} );
+// are same length
+describe( 'areSameLength(str1, str2)', function ()
```

Comment:
I wonder if this function works with 2 arrays also? maybe 2 objects? How about 2 numbers?